components/feature/syncedtabs/src/main/java/mozilla/components/feature/syncedtabs/storage/SyncedTabsStorage.kt

```diff
@@ -63,7 +64,10 @@ class SyncedTabsStorage(
                 .debounce(debounceMillis)
                 .collect { tabs ->
                     tabsStorage.store(tabs)
-                    onStoreComplete()
```

Comment:
When I went to wire it up, I realized the signature of the type was wrong. It should've been `suspend () -> Unit`, since `FxaAccountManager::syncNow` is a suspending function. Verifying that in test through Mockito would require some [additional test infrastructure](https://discuss.kotlinlang.org/t/verifying-suspending-functions-with-mockito-or-alternatives/2492/9).

@grigoryk and I had talked about it more, and we were both of the opinion that it was unlikely that this parameter was one that would be customized by a client in a way other than the implicit way that this change introduced. 

Between the two, I probably just got Friday-itis and wanted to get something merged in. Happy to switch it back if you'd like!

Reply:
We could make it a suspending lambda, but I guess this is fine as well.

Reply:
I'll keep this for now just for simplicity's sake. It's less of a breaking change to switch it to an optional parameter later, in any case.